packages/common/src/lib/common/tools.ts

```diff
@@ -1546,9 +1560,7 @@ export function getAdapterDir(adapter: string): string | null {
             adapterPath = path.join(getControllerDir(), 'node_modules', possibility);
         } else {
             try {
-                adapterPath = require.resolve(possibility, {
-                    paths: getDefaultRequireResolvePaths(module)
```

Comment:
I have not found a way to port the `paths` to esm, I hope the require resolve works anyway in the occuring scenarios.